ga4gh/backends.py

```diff
@@ -309,6 +310,109 @@ def getMetadata(self):
         return ret
 
 
+def _encode_value(value):
+    if isinstance(value, (list, tuple)):
+        return [ str(v) for v in value ]
+    else:
+        return [str(value)]
+
+
+class HtslibVariantSet(VariantSet):
+    """
+    Class representing a single variant set backed by a tabix directory.
+    """
+    def __init__(self, variantSetId, vcfPath, use_bcf=True):
+        self._variantSetId = variantSetId
+        self._created = protocol.convertDatetime(datetime.datetime.now())
+        self._chromFileMap = {}
+
+        for pattern in ['*.bcf', '*.vcf.gz']:
+            for vcfFile in glob.glob(os.path.join(vcfPath, pattern)):
+                self._addFile(vcfFile)
+
+    def _addFile(self, vcfFile):
+        varFile = pysam.BCFFile(vcfFile)
+        if not varFile.index:
+            raise Exception("VCF/BCF files must be indexed")
+        for chrom in varFile.index:
+            has_records = bool(next(varFile.fetch(chrom), None))
```

Comment:
It's up to you @bioinformed, whatever you think. Do we really need to defend against this case anyway?